packages/date/src/index.js

```diff
@@ -226,50 +201,54 @@ const formatMap = {
 	/**
 	 * Gets the days in the month.
 	 *
-	 * @param {Moment} momentDate Moment instance.
+	 * @param {Date} dateValue Date ISO string or object.
 	 *
 	 * @return {string} Formatted date.
 	 */
-	t( momentDate ) {
-		return momentDate.daysInMonth();
+	t( dateValue ) {
+		return getDaysInMonth( dateValue );
 	},
 
 	// Year
 	/**
 	 * Gets whether the current year is a leap year.
 	 *
-	 * @param {Moment} momentDate Moment instance.
+	 * @param {Date} dateValue Date ISO string or object.
 	 *
 	 * @return {string} Formatted date.
 	 */
-	L( momentDate ) {
-		return momentDate.isLeapYear() ? '1' : '0';
+	L( dateValue ) {
+		return isLeapYear( dateValue ) ? '1' : '0';
 	},
 	o: 'GGGG',
```

Comment:
Shouldn't this be `RRRR`?

Reply:
Yes, it seems it should be RRRR according to the docs of the last version https://date-fns.org/v2.16.1/docs/format. But on previous versions, it was GGGG https://date-fns.org/v1.30.1/docs/format. This type of breaking change is dangerous we should be very careful about each package version upgrade.

Reply:
As a part of applying these formatting token solutions, I'll write tests for all available formats; this way we'll be able to catch these breaking errors if they happen when updating `date-fns`.